includes/Rest/AnyCommentRestComment.php

```diff
@@ -568,7 +568,7 @@ public function create_item ( $request ) {
 		 * comment_content. See wp_handle_comment_submission().
 		 */
 		if ( $this->is_comment_empty( $prepared_comment['comment_content'] ) ) {
-			return new WP_Error( 'rest_comment_content_invalid', __( 'Comment cannot be empty.', 'anycomment' ), [ 'status' => 400 ] );
+			return new WP_Error( 'rest_comment_content_invalid', __( 'The comment area cannot be empty. Did you forget to post your thoughts?', 'anycomment' ), [ 'status' => 400 ] );
```

Comment:
This is very personal, I believe. Let leave just: 

```
The comment cannot be empty. 
```

Reply:
Revising to be less personal....

```suggestion
			return new WP_Error( 'rest_comment_content_invalid', __( 'Please enter a comment.', 'anycomment' ), [ 'status' => 400 ] );
```